<launch>
    <node pkg="visual_odometry_ros" name="mono_vo_node" type="mono_vo_node" output="screen">
        <!-- Image topicname to be subscribed -->
        <param name="topicname_image" type="string" value="/0/image_raw"/>
        
        <!-- Topicname of the groundtruth -->
        <param name="topicname_gt" type="string" value="/kitti_odometry/groundtruth"/>

        <!-- topicname of the VO pose estimation to be published -->
        <param name="topicname_pose"            type="string" value="/mono_vo/pose"/>
        <param name="topicname_trajectory"      type="string" value="/mono_vo/trajectory"/>
        <param name="topicname_map_points"      type="string" value="/mono_vo/map_points"/>

        <!-- intrinsic parameter yaml file directory -->
        <param name="directory_intrinsic" type="string" value="$(find visual_odometry_ros)/config/mono/exp_cam0.yaml"/>
    </node>
</launch>